src/ui/tools/Tags.js

```diff
@@ -56,6 +56,9 @@ ui.defaultTags = {
 	input: {
 		input: "input",
 	},
+	mapbox: {
+		div: "div",
```

Comment:
Can't use prop name **div**, it needs to be descriptive prop name and custom tag name

Reply:
Mapbox require a div element. I tested with a custom element but no rendered correctly.